nex/nodes.go

```diff
@@ -96,6 +96,7 @@ func renderNodeInfo(info *controlapi.InfoResponse, id string, full bool) {
 	cols.AddRowf("Xkey", info.PublicXKey)
 	cols.AddRow("Version", info.Version)
 	cols.AddRow("Uptime", info.Uptime)
+	cols.AddRow("Available Agents", info.AvailableAgents)
```

Comment:
in the future, this is the kind of thing that's an example of a change that should be in another PR. if this PR is about stability fixes, it should contain stability fixes. This is a control interface API change.